docs/troubleshooting.md

```diff
@@ -27,3 +27,97 @@ conda activate home-robot
+Traceback (most recent call last):
+  File "/home/jaydv/Documents/home-robot/projects/real_world_ovmm/eval_episode.py", line 14, in <module>
+    from home_robot.agent.ovmm_agent.ovmm_agent import OpenVocabManipAgent
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/agent/ovmm_agent/__init__.py", line 7, in <module>
+    from .ovmm_agent import OpenVocabManipAgent, Skill, get_skill_as_one_hot_dict
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py", line 13, in <module>
+    from home_robot.agent.objectnav_agent.objectnav_agent import ObjectNavAgent
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/agent/objectnav_agent/__init__.py", line 7, in <module>
+    from .objectnav_agent import ObjectNavAgent
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/agent/objectnav_agent/objectnav_agent.py", line 16, in <module>
+    from home_robot.mapping.instance import InstanceMemory
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/mapping/__init__.py", line 5, in <module>
+    from .voxel import SparseVoxelMap, SparseVoxelMapNavigationSpace
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/mapping/voxel/__init__.py", line 5, in <module>
+    from .planners import plan_to_frontier
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/mapping/voxel/planners.py", line 8, in <module>
+    from home_robot.mapping.voxel.voxel import SparseVoxelMap
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/mapping/voxel/voxel.py", line 21, in <module>
+    from home_robot.mapping.instance import Instance, InstanceMemory, InstanceView
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/mapping/instance/__init__.py", line 6, in <module>
+    from .instance_map import InstanceMemory
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/mapping/instance/instance_map.py", line 22, in <module>
+    from home_robot.mapping.instance.matching import (
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/mapping/instance/matching.py", line 7, in <module>
+    from home_robot.utils.bboxes_3d import (
+  File "/home/jaydv/Documents/home-robot/src/home_robot/home_robot/utils/bboxes_3d.py", line 45, in <module>
+    from pytorch3d.ops import box3d_overlap
+  File "/opt/conda/envs/home-robot/lib/python3.9/site-packages/pytorch3d/ops/__init__.py", line 7, in <module>
+    from .ball_query import ball_query
+  File "/opt/conda/envs/home-robot/lib/python3.9/site-packages/pytorch3d/ops/ball_query.py", line 10, in <module>
+    from pytorch3d import _C
+ImportError: libtorch_cuda_cu.so: cannot open shared object file: No such file or directory
+```
+
+This means there was an error installing `pytorch3d`
+```
+pip uninstall pytorch3d
+pip install --no-index --no-cache-dir pytorch3d -f https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu117_pyt201/download.html
```

Comment:
should note the python version and cuda here should match your conda environment